src/libraries/System.CodeDom/ref/System.CodeDom.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <Nullable>disable</Nullable>
```

Comment:
Do we have a tracking issue to enable nullability for projects like CodeDom or don't we plan to do so?

Reply:
Yes, https://github.com/dotnet/runtime/issues/41720